index.js

```diff
@@ -4,7 +4,33 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+ function calculator() {
+    const operator = process.argv[2]
+    let val = Number(process.argv[3])
+    
+    if (operator !== 'plus' && operator !== 'minus') { 
+        if (!!Number(operator)/1 || !operator) { 
```

Comment:
Great job here accounting for the possibility that numbers can be passed without an operator!!